app/src/main/java/com/sun_aterisk/demomvp/ui/login/LoginPresenter.kt

```diff
@@ -0,0 +1,36 @@
+package com.sun_aterisk.demomvp.ui.login
+
+import android.util.Log
+import com.sun_aterisk.demomvp.base.Result
+import com.sun_aterisk.demomvp.data.model.User
+import com.sun_aterisk.demomvp.data.source.repository.post.PostRepository
+import com.sun_aterisk.demomvp.utils.AppConstraints
+import org.json.JSONArray
+
+class LoginPresenter(val postRepository: PostRepository, val loginView: LoginContract.View) :
+    LoginContract.Presenter {
```

Comment:
viết dạng như này nhé 
Thuộc tính để private thôi 
```suggestion
class LoginPresenter(
   private val postRepository: PostRepository, 
   private val loginView: LoginContract.View
) :  LoginContract.Presenter {
```